kotlin/workflow-testing/src/main/java/com/squareup/workflow/testing/RenderTester.kt

```diff
@@ -265,9 +265,9 @@ interface RenderTester<PropsT, StateT, OutputT : Any, RenderingT> {
 /* ktlint-disable parameter-list-wrapping */
 fun <PropsT, StateT, OutputT : Any, RenderingT>
     RenderTester<PropsT, StateT, OutputT, RenderingT>.expectWorker(
-  doesSameWorkAs: Worker<OutputT>,
+  doesSameWorkAs: Worker<*>,
   key: String = "",
-  output: EmittedOutput<OutputT>? = null
+  output: EmittedOutput<Any?>? = null
```

Comment:
Why is this required?

Reply:
This is how you "fake" an output from this worker. This type is the type of the output from the worker.

Reply:
Oh duhhh.